examples/FileScan/Scan_RequestUploadURL_Post/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using PangeaCyber.Net;
+using PangeaCyber.Net.FileScan;
+using PangeaCyber.Net.Exceptions;
+using System.Text;
+using System.Security.Cryptography;
+
+
+class Program
+{
+
+    static async Task Main(string[] args)
+    {
+        try
+        {
+            // Load client config. Can create it manually with Config constructor and setters
+            var clientCfg = Config.FromEnvironment("file-scan");
+
+            // To work in async it's necessary to set `QueuedRetryEnabled` to false.
+            // When we call .scan() it will throw an AcceptedRequestException immediately if
+

```

Comment:
Should remove this blank line, because right now it is dividing this passage into two different comments which makes it confusing to read.